### DISCUSSION TITLE
Is there a way to turn of text blending

### DISCUSSION THREAD

**Discussion by Varmod** [AvaloniaUI/Avalonia#13101] [Q&A]
Hi,
I'm porting an older application to Avalonia and I noticed that the text is not as crisp as in the original.
This seems to me is because Avalonia blends the edges of the text with the background color. Is there a way to turn this off?

![textBlending](https://github.com/AvaloniaUI/Avalonia/assets/52281562/6c795376-7f04-4f25-9640-a6a9c75580e4)

Both examples have the foreground color set to Black.

**Comment by robloo** [comment]
This is ClearType on Windows. Font anti-aliasing is required in almost all applications so it's surprising to find a case where its unwanted.

You can disable it using `RenderOptions.TextRenderingMode="Alias"` see examples here: https://github.com/AvaloniaUI/Avalonia/pull/9584
